Presentation.Web/app/components/global-admin/global-admin-local-admins-list.component.ts

```diff
@@ -0,0 +1,121 @@
+﻿module Kitos.GlobalAdmin.Components {
+
+    function setupComponent(): ng.IComponentOptions {
+        return {
+            bindings: {
+                localAdmins: "<",
+                callbacks: "<"
+            },
+            controller: GlobalAdminLocalAdminListController,
+            controllerAs: "ctrl",
+            templateUrl: `app/components/global-admin/global-admin-local-admins-list.view.html`
+        };
+    }
+
+    export interface IGlobalAdminLocalAdminCallbacks {
+        removeRight(rightId: number): ng.IPromise<void>;
+    }
+
+    interface IGlobalAdminLocalAdminListController extends ng.IComponentController, Utility.KendoGrid.IGridViewAccess<ILocalAdminRow> {
+        localAdmins: Array<ILocalAdminRow>;
+        callbacks: IGlobalAdminLocalAdminCallbacks;
+    }
+
+    class GlobalAdminLocalAdminListController implements IGlobalAdminLocalAdminListController
+    {
+        localAdmins: Array<ILocalAdminRow> | null = null;
+        callbacks: IGlobalAdminLocalAdminCallbacks | null = null;
+
+        mainGrid: IKendoGrid<ILocalAdminRow>;
+        mainGridOptions: IKendoGridOptions<ILocalAdminRow>;
+
+        static $inject: string[] = ["kendoGridLauncherFactory", "$scope", "userService"];
+        constructor(
+            private readonly kendoGridLauncherFactory: Utility.KendoGrid.IKendoGridLauncherFactory,
+            private readonly $scope: ng.IScope,
+            private readonly userService: Kitos.Services.IUserService) {
+        }
+
+
+        $onInit() {
+            if (this.localAdmins === null) {
+                throw "Missing parameter 'localAdmins'";
+            }
+            if (this.callbacks === null) {
+                throw "Missing parameter 'callbacks'";
+            }
+
+            this.loadGrid();
+        }
+
+        private loadGrid() {
+            
+            this.$scope.deleteRightMethod = this.deleteLocalAdmin;
```

Comment:
I used the scope, because when I tried using the vm reference there were some issues with actually calling the method. I tried it again and it works now, so I must had been doing something wrong back then